test/integration/resourcemanager/managedresource/reconciler_test.go

```diff
@@ -254,6 +309,539 @@ var _ = Describe("ManagedResource controller tests", func() {
 			})
 		})
 	})
+
+	Context("#Resource class", func() {
```

Comment:
```suggestion
	Context("Resource class", func() {
```

 (remove those characters everywhere, those are typically only used for unit tests of a specific function)